doc/ref_arch/openstack/chapters/chapter05.md

```diff
@@ -5,16 +5,25 @@
 
 ## Table of Contents
 * [5.1 Introduction](#5.1)
-* [5.2 Core OpenStack Services APIs.](#5.2)
-* [5.3 Consolidated Set of APIs.](#5.3)
+* [5.2 Core OpenStack Services APIs](#5.2)
+* [5.3 Consolidated Set of APIs](#5.3)
 
 <a name="5.1"></a>
-## 5.1 Introduction.
-
-This chapter presents a consolidated set of OpenStack Service APIs corresponding to the ETSi Vi-Nf inetrfaces. The Openstack Pike version is used as the baseline for these APIs and CLIs. The Chapter presents the APIs for the core OpenStack services defined in Chapter 3 and later in teh Chapter a consolidated view of these and other APIs that would be of interest.
+## 5.1 Introduction
+
+This chapter presents a consolidated set of OpenStack Service APIs corresponding to the ETSi Vi-Nf interfaces. 
```

Comment:
There is no Vi-Nf interface speficied in ETSI NFV. There is an Nf-Vi, but that corresponds to the internal API-s of OpenStack between the controler and worker nodes. 
I would suggest to change this to "ETSI NFV Vi-Vnfm and Or-Vi interfaces"

Reply:
Good catch. Updated.

Reply:
Yes I agree